src/plugins/workspace/public/components/workspace_form/workspace_collaborator_table.tsx

```diff
@@ -190,15 +232,22 @@ export const WorkspaceCollaboratorTable = ({
           setSelection([]);
           modal.close();
         },
+        selections: selection,
       });
     };
 
     return (
-      <EuiButton color="danger" iconType="trash" onClick={onClick}>
-        {i18n.translate('workspace.detail.collaborator.delete', {
-          defaultMessage: 'Delete {num} collaborators',
+      <EuiButton
+        color="danger"
+        iconType="trash"
+        onClick={onClick}
+        data-test-subj="confirm-delete-button"
+      >
+        {i18n.translate('workspace.detail.collaborator.delete.button.info', {
+          defaultMessage: `Delete {num} collaborator{pluralSuffix}`,
           values: {
             num: selection.length,
+            pluralSuffix: selection.length > 1 ? 's' : '',
```

Comment:
It seems like another kind of dynamic value when using i18n. For language like Chinese, there is no concept of plural.

Reply:
Sorry for that. 🥲🥲🥲 Any recommended solutions?

Reply:
I guess we need to use `select` syntax in the defaultMessage.

https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8265/files#diff-bdc6ed432b21d5d57771a0b595dca44e147da6c6a5690ad58d25d14ca9f90991R321 for reference.

Reply:
Thanks for the reviews, already updated.